### PR TITLE
Adds typings for latlon-geohash

### PR DESCRIPTION
**Pull request by rimig**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by rimig** [comment]
Build failure is same as on master, can someone retrigger once that's resolved?

**Comment by RyanCavanaugh** [comment]
@rimig Please fix the failures indicated in the Travis CI log.

File: types/latlon-geohash/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
```
Comment (andy-ms) [inline types/latlon-geohash/tsconfig.json]: Need `"strictFunctionTypes": true,` here.

File: types/latlon-geohash/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Type definitions for latlon-geohash 1.1
+// Project: https://github.com/chrisveness/latlon-geohash
+// Definitions by: Robert Imig <https://github.com/rimig>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare module "latlon-geohash" {
```
Comment (andy-ms) [inline types/latlon-geohash/index.d.ts]: Don't use `declare module`, just export things from the top-level directly.

**Pull request merged**